src/wrap-store/wrapStore.js

```diff
@@ -130,4 +130,18 @@ export default (store, {
   } else {
     console.warn('runtime.onConnectExternal is not supported');
   }
+
+  /**
+   * Safety message to tabs for content scripts
+   */
+  chrome.tabs.query({}, tabs => {
```

Comment:
I would rather not use the tabs API since that forces users of this lib to use that API. Unfortunately, the tabs API is also the one that leads to that big scary "this extension can do anything on all pages!" warning if you set `<all_urls>`.

Reply:
Totally get that sentiment.  Try to avoid that permission as well.  Fortunately, you only need that permission if you want to access details from the tab.  That being said open to any approach that makes the most sense with the least friction.

> You can use most chrome.tabs methods and events without declaring any permissions in the extension's manifest file. However, if you require access to the url, title, or favIconUrl properties of tabs.Tab, you must declare the "tabs" permission in the manifest, as shown below:
> From: https://developer.chrome.com/extensions/tabs

Reply:
Ah got it!